circle.yml

```diff
@@ -0,0 +1,11 @@
+machine:
+  node:
+    version: 6.5.0
+
+dependencies:
+  cache_directories:
+    - "node_modules"
+test:
+  override:
+    - npm run test:ci
+    - ./node_modules/.bin/codeclimate-test-reporter < ./coverage/lcov.info
```

Comment:
since it's in this repo now it automatically works with travis, I personally prefer circle so I was going to use it in mine - I can remove this file, I'm assuming you use travis everywhere else so it doesn't make sense to break the pattern